docs/plugin_loading.md

```diff
@@ -0,0 +1,33 @@
+# Plugin Loading
+
+To load in AMP plugins, select the `Add Plugins...` option from the `Plugins` section of the upper
+taskbar.
+
+![](./pngs/add_plugins.png)
+
+Navigate to the `plugins` folder that was extracted from `plugins.zip`.  Select any of the `.plg`
+folders and hit the `Open` button to load the plugin.
```

Comment:
Probably change to "select the .plg folder corresponding to the plugin you want to load" or something like that